### DISCUSSION TITLE
Create a Single Interface for editing multiple Items

### DISCUSSION THREAD

**Discussion by mayankpunetha007** [directus/directus#10840] [Archive]
Is there a way to extend directus to create a custom collection interface consisting of key fields from multiple interfaces to make inputting  data more convenient and faster where there are multiple relationships involved?

**Comment by azrikahar** [comment]
Not sure if I'm understanding this correctly, but would something like a Spreadsheet Layout (https://github.com/directus/directus/discussions/3874) possibly achieve what you would like to do here?

**Reply by mayankpunetha007** [reply]
Hi #3874 Also sounds good.

But what I am looking is updating multiple tables from a single Item view for hirearchial relationships between tables it get cubersome to edit multiple tables by going in individually, Is there a way to create a single interface for multiple tables and manage the logics to update order of update/relationships via a custom filter/action on this combined view of items(or a single item with dummy column inputs where filters/actions can be used to customize entry to multiple tables)
containing multiple fields/entries from individual tables?

Hope that clears it up.

Something I could extend to achieve this?

**Reply by azrikahar** [reply]
Hmm that's interesting. @nickrum will there be any possible blockers for this scenario via extensions? IIRC they are resolved in the past, but just wanted to confirm with you 👍

**Reply by nickrum** [reply]
No blockers I can think of 👍 Though, there are probably a lot of different ways to implement this.
The easiest solution I can think of is to add a hook that simply "syncs" the changes done to one table with the others.
Alternatively, you could build a custom layout that somehow allows you to select the tables you want to update.
